File: tier1/csharp/b6ca245e/Report.cs
using System;
using System.Reflection;

public static class Report
{
    public static string Build(string name)
    {
        var type = Type.GetType("System.String");
        var upper = type.GetMethod("ToUpperInvariant", Type.EmptyTypes);
        return (string)upper.Invoke(name, null);
    }
}

File: tier1/java/4e458ad6/Report.java
import java.lang.reflect.Method;

public final class Report {
    public static String build(String name) {
        try {
            Class<?> strings = Class.forName("java.lang.String");
            Method upper = strings.getMethod("toUpperCase");
            return (String) upper.invoke(name);
        } catch (ReflectiveOperationException e) {
            return "";
        }
    }
}
